src/utils/DashboardHelperFunctions.js

```diff
@@ -1,40 +1,42 @@
 import axios from 'axios';
 
 export const putIncidents = (oktaAxios, incidents, status) => {
-  const reviewedIncidents = incidents.map(incident => {
-    return {
-      status: status,
-      city: incident.city,
-      state: incident.state,
-      force_rank: incident.force_rank,
```

Comment:
Hmmm so I know that this is going to cause a conflict with @austin-carman 's latest PR so I definitely recommend he review this as well, just to ensure you both agree on a way forward for this snippet that doesn't mess up functionality for either of you.

Reply:
That bit of code is pretty unecessary. All the props on the incident except for status are just being passed through, not even renamed. { ...inc, status } does the exact same thing.